src/pages/Home/Home.scss

```diff
@@ -56,34 +56,35 @@
     display: flex;
     justify-content: space-between;
     align-items: center;
-    min-height: auto;
-    flex-wrap: wrap;
-    padding: rem(0) rem(100px);
+    min-height: calc(100vh - 100px);
```

Comment:
this is much when viewing on small screen, can this be reduce?

Reply:
Done.